projects/documentation/content/guides/styling-components.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: guide.njk
+title: 'Styling Components: Spectrum Web Components'
+displayName: Styling Components
+slug: styling-components
+---
+
+# Styling Components
+
+This document outlines the details of the CSS styling for Spectrum Web Components.
+
+## Component Styling
+
+For each component in the project, there are upto 3 CSS files:
+
+-   `spectrum-[component-name].css` - This is the main CSS file for the component that is generated by the last build from the Spectrum CSS project before the cutoff.
```

Comment:
Should we explain what we mean by "the cutoff" since these docs are external-facing?

Reply:
```suggestion
-   `spectrum-[component-name].css` - This is the main CSS file for the component that is generated by the last build from the Spectrum CSS project before we removed that package as a dependency.
```

@nikkimk what about this?